src/main/resources/htmlpublisher/HtmlPublisherTarget/config.properties

```diff
@@ -7,4 +7,5 @@ reportName.title=Report title
 keepAll.title=Keep past HTML reports
 alwaysLinkToLastBuild.title=Always link to last build
 allowMissing.title=Allow missing report
-escapeUnderscores.title=Escape underscores in Report Title
\ No newline at end of file
+escapeUnderscores.title=Escape underscores in Report Title
+onlyCreateReportWithDifferentName.title=Only Create reports with different names
```

Comment:
Might be better being something like "Only allow creation of reports with different names"?

Reply:
corrected